web/src/components/common/ListingDescription.tsx

```diff
@@ -61,17 +62,23 @@ const ListingDescription: React.FC<ListingDescriptionProps> = ({
     numCommits,
     minCommits,
   },
+  priceFontSize,
+  commitProgressFontSize,
 }) => (
   <Container>
     <DeadlineTag deadline={deadline} largerFont />
     <ListingName>{name}</ListingName>
-    <ListingPrice price={price} oldPrice={oldPrice} largerFont />
+    <ListingPrice
+      price={price}
+      oldPrice={oldPrice}
+      largerFont={priceFontSize !== 'small'}
```

Comment:
I think this part is really weird, like allowing 3 options for `priceFontSize` but it really only resolves to 2 values ('large' and 'medium' has no difference).

Reply:
@hzxcaryn Okay! As discussed over chat, will only allow `priceFontSize` to accept `'medium'` or `'large'`.